packages/core/logging/core-logging-browser-internal/src/appenders/console_appender.ts

```diff
@@ -25,8 +26,7 @@ export class ConsoleAppender implements DisposableAppender {
    * @param record `LogRecord` instance to be logged.
    */
   public append(record: LogRecord) {
-    // eslint-disable-next-line no-console
-    console.log(this.layout.format(record));
```

Comment:
Ideally, we'll need a similar ES rule for calls to `kibanaConsole` (given the same rules should apply). But I guess this is a task on its own and can be done as a follow-up

Reply:
Good point. I created an issue: https://github.com/elastic/kibana/issues/176340

Reply:
Well, in the meantime I have implemented a new rule and pushed to this PR.